src/static/css/index.css

```diff
@@ -105,42 +105,29 @@ p {
   font-family: 'Poppins', sans-serif;
 }
 
-.intro-image-wrapper {
-  -ms-grid-column: 1;
-  -ms-grid-row: 2;
-  grid-area: image;
-  display: none;
-  width: 100%;
-}
-
 .intro-image {
   background-image: url(/static/images/home-hero.png);
   background-size: cover;
   height: 67.5vw;
   width: 86vw;
 }
 
-.intro-image-year-wrapper {
+.intro-image-wrapper {
   -ms-grid-column: 2;
   grid-area: image;
-  width: 100%;
-}
-
-.intro-image-year {
-  background-image: url(/static/images/home-hero.png);
-  background-size: cover;
-  max-height: 470px;
-  max-width: 458px;
-  height: 41vw;
-  width: 40vw;
+  max-height: 485px;
+  max-width: 485px;
```

Comment:
These magic numbers move it a bit too much on tablet mode (select iPad in Chrome Dev tools and compare it to production).

Reply:
The previous magic numbers were:
max-height: 470px;
max-width: 458px;
height: 41vw;
width: 40vw;

And I reduced these to the above
 max-height: 485px;
 max-width: 485px;

I found them easier to understand and to update in the future, plus the hero image looks slightly better now as it overlaps a bit with the slanted background design.

Reply:
Agree it’s great to reduce magic numbers. As I said I’m happy you’ve cleaned up a lot of CSS in this PR. And totally agree with your decision to move the logo down - you’re right it does look better.

However, did you look at iPad (landscape mode btw) in Chrome Dev tools? Seems a bit too far to the left compared to production. Noticed it first when testing real iPad but luckily also shows in Chrome in emulation mode. Though maybe I wouldn’t care if I wasn’t directly comparing it prod so maybe we should just accept this? Let me know your thoughts anyway.

Reply:
I understand what you mean. 
In this case, I think the missing bit is a margin between the main `grid` areas: intro and intro-image-wrapper.

`grid-gap: 2rem;` on `.intro-container` would solve this.